### ISSUE TITLE
Please add subdomain leasing!

### ISSUE THREAD

**Issue by chasecattaneo** [district0x/name-bazaar#13]
Subdomain leasing for ENS domains will be HUGE. Please add to Name Bazaar

**Comment by chasecattaneo** [comment]
if you are the owner of a domain like (I'm just using any example) cabosanlucas.eth you can collect rent on subdomains like hotels.cabosanlucas.eth, restaurants.cabosanlucas.eth, activities.cabosanlucas.eth ... any subdomain you can think of.. it's a win win for owners and buyers of subdomains.. owners get monthly passive income and buyers get a subdomain for less than buying the entire domain

**Comment by chasecattaneo** [comment]
there would need to be functionality in place where you can't sell the entire domain during an active leasing period

eventually any good ENS domain will be locked up just like the .coms and then subdomains will be desired

this feature gives buyers and sellers more options

**Comment by madvas** [comment]
This feature might be interesting in future, but I think there's no market for it yet. Massive public is not even properly convinced about utility of top level names yet, not mentioning subnames. 
I think good use case for subnames for now is what Status is doing, issuing usernames in their app by issuing <username>.statusim.eth names. 
So, in first versions this feature probably won't be added, but it might be later as ENS gets more popular and, more importantly, when issuing subnames gets more standardised, because at the moment it's not the case at all.

**Comment by chasecattaneo** [comment]
I agree with your point. It is very early for ENS domains in general. Please keep this feature in mind for the future.
